source/PrintData.h

```diff
@@ -0,0 +1,46 @@
+/* PrintData.h
+Copyright (c) 2022 by warp-core
+
+Main function for Endless Sky, a space exploration and combat RPG.
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+*/
+
+#ifndef PRINT_DATA_H_
+#define PRINT_DATA_H_
+
+class PrintData {
+public:
+	static void Print(char *argv[]);
+
+
+private:
+	static void Help();
+
+	static void Ships(char *argv[]);
```

Comment:
```suggestion
	static void Ships(const char * const *argv);
```

and everywhere else too

Reply:
I got an error about not being allowed to do that when I tried it before.

Reply:
That's cause you forgot the second const:

```c++
const char * const *
//           ^^^^^
```